.github/workflows/deploy-staging.yml

```
@@ -22,12 +22,37 @@ on:
+        required: false
+        default: "develop"
+        type: string
 
 jobs:
+  branch-validation:
+    name: validate branch
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)